File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.cc
#include<iostream>
using namespace std;

int gcd(int a, int b) {
    if (b == 0) return a;
    return gcd(b, a%b);
}
int main() {
    int a, b;
    cin >> a >> b;
    int div = gcd(a,b);
    int mul = div * (a/div) * (b/div);
    cout << div << "\n" << mul;
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    var answer = [];
    let div = 1;
    const size = n > m ? m : n;
    for (let i = 2; i <= size; i++) {
        if (n%i === 0 && m%i === 0) div = i;
    }
    const mul = div * (n/div) * (m/div);
    answer.push(div);
    answer.push(mul);
    return answer;
}